odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,24 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
+Non-insertable properties SHOULD be omitted from the request body.
+If they are provided, services MUST either ignore the values in the request body or fail the request,
+for example if the provided values do not match the service-determined values.
```

Comment:
```suggestion
If they are provided, services MUST either ignore the values in the request body or fail the request
if the provided values do not match the service-determined values.
```

Options:
1. check metadata and don't look at the value if it is "read-only"
2. check metadata, look at the value, compute default, compare, then reject if different